README.md

```diff
@@ -251,7 +253,7 @@ analysis results, with the enum values indicating the following:
 ### Review Comments
 
 Review comments are comments that were written by a person rather than by a
-machine. These are stored in the "refs/notes/devtools/discuss" ref, and
+machine. These are stored in the "refs/notes/devtools/comments" ref, and
```

Comment:
I was originally going to use "comments", but then worried that it was too similar to "refs/notes/commits" (the default notes ref), and would make it too easy for users (myself in particular) to make mistakes.